.vscode/tasks.json

```diff
@@ -0,0 +1,389 @@
+    },
+    {
+      "label": "Build TS WindowsGL Release",
+      "command": "dotnet",
+      "type": "process",
+      "args": [
+        "build",
+        "${workspaceFolder}/DXMainClient/DXMainClient.csproj",
+        "--configuration=Release",
+        "--framework=net7.0-windows",
+        "--property:GAME=TS",
+        "--property:ENGINE=WindowsGL",
+        "--property:GenerateFullPaths=true",
+        "--consoleloggerparameters:NoSummary"
+      ],
+      "problemMatcher": "$msCompile"
+    },
+    {
+      "label": "Build TS WindowsXNA Release",
+      "command": "dotnet",
+      "type": "process",
+      "args": [
+        "build",
+        "${workspaceFolder}/DXMainClient/DXMainClient.csproj",
+        "--configuration=Release",
+        "--framework=net7.0-windows",
+        "--property:GAME=TS",
+        "--property:ENGINE=WindowsXNA",
+        "--property:GenerateFullPaths=true",
+        "--consoleloggerparameters:NoSummary"
+      ],
+      "problemMatcher": "$msCompile"
+    },
+    {
+      "label": "Build YR UniversalGL Release",
+      "command": "dotnet",
+      "type": "process",
+      "args": [
+        "build",
+        "${workspaceFolder}/DXMainClient/DXMainClient.csproj",
+        "--configuration=Release",
+        "--framework=net7.0",
+        "--property:GAME=YR",
+        "--property:ENGINE=UniversalGL",
+        "--property:GenerateFullPaths=true",
+        "--consoleloggerparameters:NoSummary"
+      ],
+      "problemMatcher": "$msCompile"
+    },
+    {
+      "label": "Build YR WindowsDX Release",
+      "command": "dotnet",
+      "type": "process",
+      "args": [
+        "build",
+        "${workspaceFolder}/DXMainClient/DXMainClient.csproj",
+        "--configuration=Release",
+        "--framework=net7.0-windows",
+        "--property:GAME=YR",
+        "--property:ENGINE=WindowsDX",
+        "--property:GenerateFullPaths=true",
+        "--consoleloggerparameters:NoSummary"
+      ],
+      "problemMatcher": "$msCompile"
+    },
+    {
+      "label": "Build YR WindowsGL Release",
+      "command": "dotnet",
+      "type": "process",
+      "args": [
+        "build",
+        "${workspaceFolder}/DXMainClient/DXMainClient.csproj",
+        "--configuration=Release",
+        "--framework=net7.0-windows",
+        "--property:GAME=YR",
+        "--property:ENGINE=WindowsGL",
+        "--property:GenerateFullPaths=true",
+        "--consoleloggerparameters:NoSummary"
+      ],
+      "problemMatcher": "$msCompile"
+    },
+    {
+      "label": "Build YR WindowsXNA Release",
+      "command": "dotnet",
+      "type": "process",
+      "args": [
+        "build",
+        "${workspaceFolder}/DXMainClient/DXMainClient.csproj",
+        "--configuration=Release",
+        "--framework=net7.0-windows",
+        "--property:GAME=YR",
+        "--property:ENGINE=WindowsXNA",
+        "--property:GenerateFullPaths=true",
+        "--consoleloggerparameters:NoSummary"
+      ],
```

Comment:
XNA builds crash on launch because platform `x86` and 32bit dotnet.exe are not set. Also see https://github.com/CnCNet/xna-cncnet-client/blob/develop/Directory.Build.targets.